R/derive_joined.R

```diff
@@ -63,6 +63,16 @@
 #'
 #'   *Permitted Values*: list of variables or named expressions created by `exprs()`
 #'
+#' @param tmp_obs_nr_var Temporary observation number
```

Comment:
Do we have tests for this argument?  I couldn't find any

Reply:
Also...would it be handy to provide the observation numbers to the user?   in case you want to inspect to see if something seems amiss???

Reply:
We have 100% coverage. In `derive_vars_joined()` `tmp_obs_nr_var` is just passed on to `get_joined_data()`. `tmp_obs_nr_var` in `get_joined_data()` is tested via a test case for `derive_var_joined_exist_flag()`.

If you want to include the temporary observation number in the output dataset, you can add the variable to `new_vars`.

Reply:
awesome!!  just checking.   do we mention anywhere that we can use `new_vars` to grab the temps?  I think this would be helpful for folks.

Reply:
Agree, I have added it to the description of `tmp_obs_nr_var`.

Reply:
Thanks!!!  what about adding the text to the other functions using `tmp_obs_nr_var`?

Reply:
`filter_joined()` and `derive_var_joined_exist_flag()` do not provide a `new_vars` argument. Thus it makes no sense to add the text there.